### PR TITLE
430

### PR DESCRIPTION
**Pull request by iChris**

### CODE REVIEW

**Comment by iChris** [comment]
Requested a review on this one @wesbos just to make sure it's all good since it'll likely get a few eyeballs on it. :)

File: shows/430 - Creator of Wordle.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Sanity.io is a real-time headless CMS with a fully customizable Content Studio built in React. Get a Sanity powered site up and running in minutes at [sanity.io/create](https://www.sanity.io/create). Get an awesome supercharged free developer plan on [sanity.io/syntax](https://www.sanity.io/syntax).
+
+## Freshbooks - Sponsor
+
+Get a 30 day free trial of Freshbooks at [[freshbooks.com/syntax](http://freshbooks.com/syntax)]([https://freshbooks.com/syntax](https://freshbooks.com/syntax))
+
+FreshBooks is made for non-accountants like you. It’s an intuitive, easy-to-use accounting and invoicing software that does it all—from sending and following up on invoices, to automatically capturing receipt data for expenses, to getting you paid twice as fast.
+
+This episode is brought to you by FreshBooks, the invoicing and accounting solution that’s built for owners—and their clients.
```
Comment (wesbos) [inline shows/430 - Creator of Wordle.md:24]: take the below freshbooks part out- this is the verbal ad read part - doesn't need to be in the show notes

File: shows/430 - Creator of Wordle.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+number: 430
+title: Creator of Wordle Josh Wardle
```
Comment (wesbos) [inline shows/430 - Creator of Wordle.md:3]: Can you make this Creator of Wordle - Josh Wardle

**Review by wesbos — COMMENTED** [review]
Just a few small edits

File: shows/430 - Creator of Wordle.md (hunk 2 above)
Comment (iChris) [reply shows/430 - Creator of Wordle.md]: Done!

File: shows/430 - Creator of Wordle.md (hunk 1 above)
Comment (iChris) [reply shows/430 - Creator of Wordle.md]: Done!

**Pull request merged**